packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.types.tsx

```diff
@@ -0,0 +1,102 @@
+import { InterfacePropertyType, PropertyType } from '../../utilities/parser/index';
+
+export interface IApiReferencesTableSetProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * A json object to populate the table
+   */
+  jsonDocs?: IPageJson;
+}
+
+export interface ITokenJson {
+  text: string;
+  hyperlinkedPage?: string;
+}
+
+export interface ITableRowJson {
+  name: string;
+  typeTokens: ITokenJson[];
+  defaultValue?: string;
+  descriptionHtml: string;
+  deprecated: boolean;
+  deprecatedMessage?: string;
+  kind?: 'Method' | 'Property';
+}
+
+export interface IEnumTableRowJson {
+  name: string;
+  descriptionHtml: string;
+  value: string;
+}
+
+export interface ITableJson {
+  kind: 'interface' | 'enum' | 'class';
+  name: string;
+  extendsTokens: ITokenJson[];
+  descriptionHtml: string;
+  members: ITableRowJson[] | IEnumTableRowJson[];
+}
+
+export interface IPageJson {
+  tables: ITableJson[];
+  name: string;
+}
+
+export interface IApiProperty {
+  propertyName: string;
+  extends?: string;
+  extendsTokens?: ILinkToken[];
+  description?: string;
+  title: string;
+  propertyType: PropertyType;
+  property: IApiInterfaceProperty[] | IApiEnumProperty[];
+  methods?: IMethod[];
+}
+
+/**
+ * Used to keep track of where the page will live on the site
+ */
+export enum PageKind {
```

Comment:
Does this need to be an enum or can it just be a type? `export type PageKind = 'References' | 'Components'`